pkg/build/buildopts.go

```diff
@@ -131,6 +133,8 @@ func NewBuildOpts(ctx context.Context, basePath string, contextMap map[string][]
 		ctxDir = c
 	}
 
+	bp := utils.BuildPlatforms()
+
```

Comment:
Thanks for the fix @stevapple! I do think we can just use DefaultSpec here unless @wlan0 has any concerns with that.